### DISCUSSION TITLE
Reduce unused space in core modals

### DISCUSSION THREAD

**Discussion by grahamford** [vmware/clarity#6398] [Ideas] [answered]
We open our modal component programatically and generate the content components on the fly. These child components are output within the `cds-modal-content` tags. 

Because of this we don't use the `cds-modal-actions` section. The output on screen has an area of whitespace for these actions, even though it's empty. The clr version of the modal doesn't exhibit this problem. Do you think it would be possible to skip unused sections in the modal component to remove this whitespace below the content area in the cds version?

**Comment by mathisscott** [comment, marked as answer]
This will be addressed when we merge the dropdowns/popups in the PR associated with issue #5394.  Please keep an eye on that issue.

The fix will be available when `@cds/core` v6.0 is released.
